### PR TITLE
Set up testing environment for API

### PR DESCRIPTION
**Pull request by Anastasiachyp**
PT chore: https://www.pivotaltracker.com/story/show/181399293

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -17,7 +17,9 @@ group :development, :test do
   gem 'coveralls', require: false
   gem 'pry-rails'
   gem 'pry-byebug'
+  gem 'rspec'
```
Comment (tochman) [inline Gemfile]: Why do you add this gem? There's already a gem called `rspec-rails` in the Gemfile

File: Gemfile [hunk 2]
```diff
@@ -17,7 +17,9 @@ group :development, :test do
...
   gem 'rspec-rails'
+  gem 'rspec-rails-matchers'
```
Comment (tochman) [inline Gemfile]: What does this gem do for you? Why is it included?

File: db/schema.rb [hunk 3]
```diff
@@ -10,9 +10,16 @@
...
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "products", force: :cascade do |t|
+    t.string "name"
+    t.float "price"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+  end
```
Comment (tochman) [inline db/schema.rb]: Why do you have this in your schema?

File: Gemfile (hunk 1 above)
Comment (Anastasiachyp) [reply Gemfile]: I will remove it

File: Gemfile (hunk 2 above)
Comment (Anastasiachyp) [reply Gemfile]: The rspec gem was already there before, I thought something was missing. I don't understand which gem is missing

File: db/schema.rb (hunk 3 above)
Comment (Anastasiachyp) [reply db/schema.rb]: It's not mine, It was already before

**Pull request closed without merging**